guides/doc-Release_Notes/topics/foreman-3.3.1.adoc

```diff
@@ -0,0 +1,78 @@
+* ks= kernel parameter in Kickstart default iPXE causes RHEL9 Anaconda failure to start - https://projects.theforeman.org/issues/34975[#34975]
+* Allow setting up rex pull during provisioning and registration - https://projects.theforeman.org/issues/34969[#34969]
+* Ubuntu Autoinstall PXELinux template does not reference Smart Proxy correctly - https://projects.theforeman.org/issues/34941[#34941]
+
+=== Compute resources - VMware
+
+* hammer cannot use the cluster name or id as valid input when clusters are residing inside folders and fails with error fog not found - https://projects.theforeman.org/issues/35438[#35438]
+
+=== Tests
+
+* Update show_me_the_cookies to 6.x - https://projects.theforeman.org/issues/35371[#35371]
+
+=== Settings
+
+* Ignore calico interfaces from Foreman - https://projects.theforeman.org/issues/35289[#35289]
+
+=== Host creation
+
+* Bulk rebuild hosts can not reset the build token - https://projects.theforeman.org/issues/35203[#35203]
+
+=== Unattended installations
+
+* Add installation medium for Stream 9 - https://projects.theforeman.org/issues/35151[#35151]
+
+=== Audit Log
+
+* Audit page shows "auditable id / Host2" for "Host1" but Host2 does not exist or deleted from the all hosts - https://projects.theforeman.org/issues/35132[#35132]
+
+=== Inventory
+
+* Hide reports "Origin" column when table is already filtered by origin - https://projects.theforeman.org/issues/35126[#35126]
+* Host\'s power status ping is limited to three seconds - https://projects.theforeman.org/issues/35083[#35083]
+
+=== Host registration
+
+* In Global Registration, using another LANG like pt_BR.UTF-8 breaks the UUID - https://projects.theforeman.org/issues/35051[#35051]
+
+=== Smart Proxy
+
+* Smart Proxy upgrade/install fails due to proxy configuration in  \'HTTP(S) proxy\' in settings - https://projects.theforeman.org/issues/34996[#34996]
+
+=== Web Interface
+
+* Models index page does 3 same API requests  - https://projects.theforeman.org/issues/34987[#34987]
+
+=== Organizations and Locations
+
+* Taxonomies are being set during extlogin too early, leading to user\'s default taxonomies being ignored - https://projects.theforeman.org/issues/34949[#34949]
+
+=== Rake tasks
+
+* \`foreman-rake permissions:reset\` should show a warning that it\'s required to manually update the password in \'foreman.yml\' - https://projects.theforeman.org/issues/31966[#31966]
+
+== Installer
+
+=== External modules
+
+* SSO feature settings are not getting enabled and failed with HTTPD CONF issue - https://projects.theforeman.org/issues/35524[#35524]
+* cache-enabled setting for proxy content module isn\'t migrated to \'true\' properly - https://projects.theforeman.org/issues/35157[#35157]
+
+== Packaging
+
+* Sendmail package not present on RHEL8 and needs manual configuration - https://projects.theforeman.org/issues/35461[#35461]
+
+== Smart Proxy
+
```

Comment:
I would like no note that Smart Proxy section is already included above, unless this is here on purpose as a higher level title.

Reply:
There is: it reflects the structure in Redmine. So level x.y is the project (https://projects.theforeman.org/projects) and x.y.z is the category. The project Foreman has a category Smart Proxy, so `4.1.10 Smart Proxy` reflects that category. `4.4 Smart Proxy` refers to the Smart Proxy Project. `4.4.1 Packaging` also refers to the Packaging category where `4.3 Packaging` refers to the Packaging project.

Reply:
Thanks for clarification :+1: